Modules/IndividualAssessment/classes/class.ilIndividualAssessmentMemberGUI.php

```diff
@@ -372,7 +372,7 @@ protected function initGradingForm($may_be_edited = true, $amend = false) {
 		$form->addItem($txt);
 
 		$date = new ilDateTimeInputGUI($this->lng->txt('iass_event_time'), 'event_time');
-		$date->setShowTime(false);
+		$date->setShowTime(true);
```

Comment:
Please fix this on the presentation layer. We do not want to add the possibility to tell the time of the examination, but instead just remove the time in the presentation of the date.